docs/source/markdown/podman.1.md

```diff
@@ -440,6 +440,8 @@ Or just add the content manually.
 
 See the `subuid(5)` and `subgid(5)` man pages for more information.
 
+Note: Extra whitespace in the /etc/subuid,/etc/subgid files can cause no entry failures, even if an entry is present.
```

Comment:
Slight preference for `/etc/subuid and /etc/subgid` instead of a comma list. (If you stick with comma, please add a space).

Some unclarity with "cause no entry failures", I wonder if backticks around `no entry` might help?

Finally, could you give an example of what you mean by "extra whitespace"? Is it around the colons? Can trailing blanks cause problems?

Reply:
Fails with trailing whitespace, is the one I know of

Reply:
Okay, how about this:
>Note: whitespace in any row of `/etc/subuid` or `/etc/subgid`, including trailing blanks, may result in `no entry` failures.

Reply:
Done